### PR TITLE
Changed the long hover to hover at the drone's current altitude

### PR DESCRIPTION
**Pull request by jozgd**
Now the drone will long hover at its current altitude when it runs out of tasks

### CODE REVIEW

**Review by ClayMav — CHANGES_REQUESTED** [review]
I thought we had discussed being able to create hover tasks with no altitude and they would default to the current altitude. Then this line would just not include a height.

**Comment by Thomas-McKanna** [comment]
I know I told you this would work, but I wasn't thinking of the larger picture. Clay is right in that if the hover task is given no altitude it should default to its current altitude. There could be instances other than when we run out of tasks where we want to make a hover task that remains at current altitude, and we don't want to have to pass in the current altitude as a parameter each time.

I would make altitude an optional argument that defaults to None and then grab the current altitude at the first iteration of perform.

**Review by christopher-o-toole — DISMISSED** [review]
Looks fine after the latest commit.

File: flight/drone/drone_controller.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def run(self):
             sleep(c.DELAY_INTERVAL)  # Sleep in case was doing write operation
             self._splitter.exit()
 
-    def add_hover_task(self, altitude, duration, priority=c.Priorities.LOW):
+    def add_hover_task(self, duration=c.DEFAULT_HOVER_DURATION, altitude=None, priority=c.Priorities.LOW):
```
Comment (ClayMav) [inline flight/drone/drone_controller.py:144]: Just default altitude to be rangefinder distance, since that's the only checking you are doing right now

**Review by ClayMav — CHANGES_REQUESTED** [review]

**Comment by ClayMav** [comment]
@jozgd Can we get this finished up as well?

**Comment by ClayMav** [comment]
Fixes #55

**Review by Thomas-McKanna — APPROVED** [review]
I have gone through the code with John and we tested it in the simulator. It seems to be working (we did notice some anomalies in the simulator, such as occasionally drifting slowly downward). I think that this weird behavior was just the simulator acting up, since the changed lines of code in this pull request do not affect changing the altitude, only setting a target altitude.

**Review by ClayMav — APPROVED** [review]

**Pull request merged**